backend/main/test/Shared/Adapters/NishikiTableClient.test.ts

```diff
@@ -76,4 +76,30 @@ describe.sequential("groups operation", () => {
 			expect(result).toEqual(expectedGroup);
 		}
 	});
+
+	describe("get a list of users who belong to the requested group", () => {
+		it("there are users belonging to group", async () => {
+			const containsUsersGroup = groupData.groupData[0];
+
+			const usersIds = await nishikiClient.listOfUsersInGroup(
+				containsUsersGroup.groupId,
+			);
+
+			expect(usersIds.length).toBe(2);
```

Comment:
@hitohata 
When I saw the number `2, ' it took a little time for me to understand its meaning. So, I prefer using original data as an expected value. But it's my preference so I leave it to you whether you change it. 👍 

```suggestion
			expect(usersIds.length).toBe(containsUsersGroup.users?.length);
```